lib/cogctl/actions/relays/delete.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(options, args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, args, nil, &1))
+  end
+  def run(_options, [], _config, _endpoint), do: display_arguments_error
+  def run(_options, args, _config, endpoint) do
+    do_delete(endpoint, args)
+  end
+
+  defp do_delete(endpoint, relay_names) do
+    Enum.each(relay_names, fn relay_name ->
```

Comment:
`Enum.each/2` doesn't care what the return value of the individual function invocations is, so the `{:cont, :ok}` and `{:cont, :error}` bits are just noise.

Thinking a bit more, we'll need to know whether something failed or not in order to affect the exit code. As it is, this will always exit with a `0`.